### PR TITLE
docs: improve README examples of `stats/iter` namespace .

### LINKED ISSUE #1652: [RFC]: improve README examples of `stats/iter` namespace
### Description

This RFC proposes to improve the [README examples](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/iter#examples) of the `stats/iter` namespace package. Currently, namespace package READMEs only include minimal examples which are not particularly informative and do not showcase namespace functionality.

Historically, namespace examples were not given much effort due to the rapidly changing nature of the project; however, now that development and organization has settled, it would be good to revisit these packages and add proper examples to better communicate to users how to use namespace contents.

For those wanting to contribute to this issue, you can use the following steps:

- Browse through the [contents](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/iter) of the `stats/iter` namespace and identify interesting functions and packages to showcase.
- Update the example code block in the "Examples" section of the README for the `stats/iter` namespace and open a PR.
- When updating the README, examples should follow existing project conventions (e.g., ES5).
- Examples may `require` functionality from other packages to help make examples more interesting.

### Related Issues

None.

### Questions

No.

### Other

No.

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `RFC:`.

### PR DESCRIPTION
**Pull request by kumarsuraj212003**
Resolves #1652.

## Description

> What is the purpose of this pull request?

This pull request:

-   a
-   b
-   c

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #1652
-   fixes #

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/stats/iter/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 @license Apache-2.0
 
-Copyright (c) 2018 The Stdlib Authors.
+Copyright (c) 2024 The Stdlib Authors.
```
Comment (steff456) [inline lib/node_modules/@stdlib/stats/iter/README.md]:
```suggestion
Copyright (c) 2018 The Stdlib Authors.
```

We only define the copyright year when we create the file

File: lib/node_modules/@stdlib/stats/iter/README.md [hunk 2]
```diff
@@ -108,6 +108,24 @@ var getKeys = require( '@stdlib/utils/keys' );
...
+// returns 18.0
+
+// Compute the maximum temperature
+var maxTemperature = ns.itermax( temperatures );
+// returns 26.0
+
+// Compute the arithmetic mean temperature
+var meanTemperature = ns.itermean( temperatures );
+// returns 21.8
+
```
Comment (steff456) [inline lib/node_modules/@stdlib/stats/iter/README.md]:
```suggestion
```

**Review by steff456 — CHANGES_REQUESTED** [review]
Thanks @kumarsuraj212003 for opening this PR!

I left two comments and please check the failing CI for the example you added.

Also, please add the requested information in the description of this PR, specially the first section describing what this PR is doing.

**Review by kumarsuraj212003 — COMMENTED** [review]
i follow stuggested changes

**Comment by kumarsuraj212003** [comment]
Screenshot (12)
can u please tell me why am i getting this error.
i have setup stdlib locally i also varified by running benchmarks and tests ,it woks smoothly

**Comment by kumarsuraj212003** [comment]
I think i should update index.js file too in `stats/iter/examples` to pass all CI

**Comment by steff456** [comment]
Hi @kumarsuraj212003, that error indicate that when the example is running, it is returning `null` instead of the expected value. Can you please check if everything is running smoothly locally by running the lint command `make lint-markdown-files FILES="<Type here the path to the readme file>"`? You can double check if everything is importing correctly as part of the package.

**Pull request closed without merging**